static/app/components/u2f/u2finterface.tsx

```diff
@@ -111,18 +126,49 @@ class U2fInterface extends React.Component<Props, State> {
       });
   }
 
+  webAuthnSignIn(authRequest) {
+    const credentials = [] as any;
+
+    authRequest.forEach(device => {
+      credentials.push({
+        id: base64urlToBuffer(device.keyHandle),
+        type: 'public-key',
+        transports: ['usb', 'ble', 'nfc'],
+      });
+    });
+
+    const publicKeyCredentialRequestOptions = {
+      challenge: base64urlToBuffer(authRequest[0].challenge),
+      allowCredentials: credentials,
+      userVerification: 'discouraged',
+      extensions: {
+        appid: authRequest[0].appId,
+      },
+    } as PublicKeyCredentialRequestOptions;
+
+    const promise = navigator.credentials.get({
+      publicKey: publicKeyCredentialRequestOptions,
+    });
+    this.submitU2fResponse(promise);
+  }
+
   invokeU2fFlow() {
```

Comment:
should we rename this now that u2f isn't what we're using in the new flow?

Reply:
resolved per conversation